pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -1,26 +1,21 @@
-{ _7zz
-, buildDotnetModule
-, copyDesktopItems
-, desktop-file-utils
-, dotnetCorePackages
-, fetchFromGitHub
-, fontconfig
-, lib
-, libICE
-, libSM
-, libX11
-, nexusmods-app
-, runCommand
-, enableUnfree ? false # Set to true to support RAR format mods
+{
+  lib,
```

Comment:
Did `nixfmt` change this ordering? It seems weird having only `lib` on top.

Reply:
No, that was manual.

I sorted almost all args, but decided to keep `lib` and our "custom" args (pname) unsorted as (in my head) they are distinct.

I can sort all args or all auto-args if you prefer, but is seems like a minor point.

Reply:
I agree `pname` should be last, which is why I didn't mention it. It is a minor point, but having to read through a big list of parameters when I could just scan it alphabetically is tedious.

Reply:
I'll fix this tomorrow, heading to sleep now.

Thanks for your feedback!

Reply:
I've reverted to everything other than `pname` being sorted alphanumerically